pkg/apis/cr/v1alpha1/types.go

```diff
@@ -178,12 +172,6 @@ type ActionSetList struct {
 	Items           []*ActionSet `json:"items"`
 }
 
-// These names are used to query Blueprint API objects.
-const (
-	BlueprintResourceName       = "blueprint"
-	BlueprintResourceNamePlural = "blueprints"
-)
-
```

Comment:
I have moved these to `consts` package to resolve import cycle issue that we faced when we tried to use them from `customresource` package.